projboard/templates/landing/homepage.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %} {% load static %} {% block content %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
```

Comment:
if you already did the extend from the base=> you can just keep the body element ( even just the content of it )

Reply:
@Anas-mhameed Yes I agree, I just made it empty html file to change it later

Reply:
If you have plans for later,
Please open an issue, and add link to this thread

Reply:
@Yarboa Done

Reply:
Please paste the link here

Reply:
@rawadabu any reply here?

Reply:
https://github.com/redhat-beyond/Smarticle/issues/74

Reply:
@Yarboa Done